ospf6d/ospf6_top.h

```diff
@@ -152,4 +157,7 @@ struct ospf6 *ospf6_lookup_by_vrf_id(vrf_id_t vrf_id);
 struct ospf6 *ospf6_lookup_by_vrf_name(const char *name);
 const char *ospf6_vrf_id_to_name(vrf_id_t vrf_id);
 
+void ospf6_restart_spf(struct ospf6 *ospf6);
```

Comment:
Why are we making these public functions?  Just order the functions as needed in ospf6_top.c and make them static

Reply:
Fixed.